README.md

```diff
@@ -1,59 +1,74 @@
-# \<link[rel="lazy-import"]\> [![Build Status](https://img.shields.io/travis/Polymer/lazy-imports.svg?style=flat-square)](https://travis-ci.org/Polymer/lazy-imports)
+# \<link rel="lazy-import"\> 
 
-Declarative lazy HTML imports
+[![Build Status](https://img.shields.io/travis/Polymer/lazy-imports.svg?style=flat-square)](https://travis-ci.org/Polymer/lazy-imports)
+
+This repo implements declarative, lazy, HTML Imports. Normal HTML Imports are eager, meaning that they are loaded and evaluated in order first, before any code that follows. There is also a [javascript API](https://www.html5rocks.com/en/tutorials/webcomponents/imports/) for performing HTML Imports, allowing lazy loading of code at any point in time. Lazy loading lets you build an app in a sharded way, so that only the minimal amount of code is loaded to display the current view. This is a key piece of [the PRPL pattern](https://developers.google.com/web/fundamentals/performance/prpl-pattern/).
```

Comment:
So, if we're talking about dynamically creating and adding a link element, we should be more explicit here because it's hella hard to find a reference to that in the article—it's there, the closest anchor I can find is https://www.html5rocks.com/en/tutorials/webcomponents/imports/#perf-parsing

In any case, should be clear here we're talking about the normal state, so recommend:
1. New paragraph before "Normal HTML imports..." so that whole para is on the same topic (state of the world without lazy-imports).
2.  Make it clear where the API is coming from. "You can dynamically load an import by adding a link element to your document (as described in HTML Imports: #include for the web(link)). The `Polymer.importHref`(link) function lets you do this in a single call. 

... or something like that?